warehouse/ingest-core/src/test/java/datawave/ingest/data/config/CachingFieldConfigHelperTest.java

```diff
@@ -0,0 +1,99 @@
+package datawave.ingest.data.config;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class CachingFieldConfigHelperTest {
+    @SuppressWarnings("unchecked")
+    @Test
+    public void testCachingBehaviorWillCallBaseMethods() {
+        // @formatter:off
+        Stream.of(new Object[][] {
+            new Object[] {
```

Comment:
Are we actually calling all these pairs? 

You are iterating over an array of arrays but there's only 1 inner array with 12 elements in it. 
It seems like we want 6 inner arrays, each with two elements {action, verify}
I think we're actually only testing the isIndexOnlyField pair here.

Reply:
It also has me wondering if maybe it's just easier to do the manual calls in the test instead of dealing with all the method references and such.

```
cachedHelper.isIndexOnlyField(fieldName);
verify(mockHelper).isIndexOnlyField(fieldName);

cachedHelper.isIndexedField(fieldName);
verify(mockHelper).isIndexedField(fieldName);

// ...
```

Reply:
That is a bug, thank you for catching that. In regards to the manual tests - I updated, thank you.